### ISSUE TITLE
Carousel Next/Previous buttons issues

### ISSUE THREAD

**Issue by JWandrocke** [SAP/ui5-webcomponents#5125]
Hi
I upgraded to 1.3.0 version and now I can not tab to the Carousel Next/previous navigation buttons. Also the Focus indicator on the button is cut off.

![image](https://user-images.githubusercontent.com/54148499/165340076-f13bea33-3642-4292-8195-0c3c8f92a67f.png)

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

The issue is reproducible in the playground samples:
https://sap.github.io/ui5-webcomponents/master/playground/components/Carousel/
with the same behavior as described by @JWandrocke.
The navigation arrow buttons aren't reachable via keyboard.
Further more focus gets cut if we use mouse press to trigger the navigation button.

Could you please check what should be the expected behavior and provide a fix for the cut focus outline.
The focus outline should be checked in all supported themes.

Best Regards,
Boyan Rakilovski

**Comment by gmkv** [comment]
Hi @JWandrocke 
Thanks for raising this issue.
Currently the carousel has inconsistent behaviour when it comes to these buttons.
The carousel has been intentionally specified to have those navigation buttons not reachable using the keyboard.
The way to change slides on the carousel is to use the Arrow keys.

I've prepared a change after which, the navigation buttons will not remain focused after being clicked, which will align the current implementation with the spec. That should avoid the confusing behavior when pressing the navigation buttons.
The squared focus outline will also be fixed.

Regards,
Georgi
